index.html

```diff
@@ -148,28 +148,29 @@ <h2 class="pt-3" id="info">What <em>is</em> Standard?</h2>
           href="https://magic.wizards.com/en/content/standard-formats-magic-gathering" rel="noopener">
             <b>Standard</b>
           </a>
-          is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're released and
-          drop out about twenty-one months later.
+          is a tournament format which consists of a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter
+          the format when they're released and drop out about twenty-one months later.
           </p>
           <p>
-          Generally Standard will grow until it would receive its ninth set, then instead it drops its oldest four sets
-          to receive it as a fifth set. There are exceptions to this. It's confusing.
+          Generally it grows until it contains eight sets. Then — with the next new joining release — the oldest four sets
+          are dropped as group and the new one is added as fifth legal set.<br>
+          There are exceptions to this. It's confusing.
           </p>
           <p>
-          A <b>Standard card</b> is a card printed or reprinted into a set currently in Standard.
+          A <b>Standard card</b> is a card printed or reprinted in a set currently part of the legal pool.
```

Comment:
While we're here do you think we should simplify this more to "A **Standard card** is a card in a set currently part of the legal pool." and massage the tooltip a bit to explain the reprints bit from scratch?

Reply:
I like that! Done.